jobs/ci-kubernetes-e2e-gce-statefulset.sh

```diff
@@ -31,8 +31,8 @@ export CLOUDSDK_CORE_PRINT_UNHANDLED_TRACEBACKS="1"
 # expected empty
 
 ### job-env
-export GINKGO_TEST_ARGS="--ginkgo.focus=\[Feature:PetSet\]"
-export PROJECT="kubernetes-petset"
+export GINKGO_TEST_ARGS="--ginkgo.focus=\[Feature:StatefulSet\]"
+export PROJECT="kubernetes-statefulset"
```

Comment:
Yes as long as this GCP project exists and is configured correctly... I would recommend leaving the projects alone for now.